tgui/src/interfaces/ntos_supermatter_monitor.ract

```diff
@@ -0,0 +1,70 @@
+
+<script>
+  component.exports = {
+    data: {
+	  integState(charge) {
+	    let max = 100
+	    if(charge == max) return 'good'
+		else if(charge > (max/2)) return 'average'
+		else return 'bad'
+	    }
+      }
+    }
+
+  component.exports = {
```

Comment:
you don't need a second set of ```component.exports``` and ```data```, just put ```BigState(val, aver, max)``` below ```integState```

Like I did here: https://github.com/tgstation/tgstation/blob/eace4eec148e8a366ee2fe6673f852b59a2e77b1/tgui/src/interfaces/mech_bay_power_console.ract#L2